purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1570,10 +1610,12 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
         MPARTICLE(6)
     }
 
-    internal data class AttributionData(
-        val data: JSONObject,
-        val network: AttributionNetwork,
-        val networkUserId: String?
-    )
     // endregion
 }
+
+internal fun Purchases.AttributionNetwork.convert(): AttributionNetwork {
+    AttributionNetwork.values().forEach { network ->
+        if (network.serverValue == this.serverValue) return network
+    }
+    return AttributionNetwork.ADJUST
+}
```

Comment:
this trick is used to convert from the `Purchases.AttributionNetwork` to the `AttributionNetwork` inside the `common` module. I am returning `ADJUST` by default because I don't want to make it nullable. What do you think

Reply:
it might be better to crash by default, and have extensive tests so that if something gets incorrectly mapped we find out right away

Reply:
I had to do a similar thing for `RCIntroEligibility`: 
- https://github.com/RevenueCat/purchases-ios/blob/develop/Purchases/Public/RCIntroEligibility.m
- https://github.com/RevenueCat/purchases-ios/blob/develop/PurchasesTests/Purchasing/RCIntroEligibilityTests.swift